version.py

```diff
@@ -40,18 +40,27 @@
 
 
 def call_git_describe(prefix='', cwd='.'):
-    cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    version_name_cmd = 'git describe --tags --match %s[0-9]*' % prefix
+    last_tag_cmd = 'git describe --tags --abbrev=0'
     try:
         version = subprocess.run(
-            cmd.split(),
+            version_name_cmd.split(),
             stdout=subprocess.PIPE,
             check=True,
         ).stdout.decode()
+
+        last_tag = subprocess.run(
+            last_tag_cmd.split(),
+            stdout=subprocess.PIPE,
+            check=True,
+        ).stdout.decode()
+
         version = version.strip()[len(prefix):]
-        if '-' in version:
-            version = '{}+dev{}.{}'.format(*version.split('-'))
+        last_tag = last_tag.strip()[len(prefix):]
```

Comment:
This won't work correctly if `prefix` contains leading whitespace. I'd add an `assert` against that.

Actually, given that `prefix` gets embedded in a bash command above, the function won't work if it contains any whitespace - leading, trailing or spaces between words.

Reply:
Done